__tests__/src/ErrorPlugin.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable class-methods-use-this, no-unused-vars, import/no-unresolved, import/extensions */
 import {
   PluginLoadOptions, Event, Properties, Plugin, ValidationResponse,
-} from '../../packages/sdk/lib';
+} from '../../packages/sdk';
```

Comment:
Not sure what happened here. The tests broke after merging in the ESM changes from master. This fixed the issue.

Reply:
ok